### DISCUSSION TITLE
Expandable group

### DISCUSSION THREAD

**Discussion by alexanderlindstrom** [home-assistant/frontend#18852] [Automations / scripts / scenes]
I hope this is the right category for this. I want to suggest a button for expanding groups in under target, see screenshots/mockups

![actions 1](https://github.com/home-assistant/frontend/assets/1754686/46991e88-9240-485e-abff-5060b4a3240b)
**This is how it normally look**

![actions 2](https://github.com/home-assistant/frontend/assets/1754686/793f7ca1-3be4-40ff-b289-aa5e79eff254)
**I want an icon (see next to the X) for expanding the group, effectively removing the group and adding all entities that are in the group**

![actions 3](https://github.com/home-assistant/frontend/assets/1754686/15b297e5-3e35-471f-a808-996dd57b19ff)
**This is what it would look like after clicking on the icon**

**Comment by ildar170975** [comment]
In this case it would be great to have a possibility to collapse a group:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/8c014715-e59f-4af3-a8a5-3c09218f24d6)

where the "expanded" group still should be shown - with a control to "collapse" and probably of a slightly different color (to show "i am expanded").
And a user can remove the "expanded" group - then all "member" entities from this group should also be removed.
Also, a user can remove a "member" entity - then the "expanded" group should be removed as well (not affecting other "member" entities).

**Comment by alexanderlindstrom** [comment]
So kind of like, "Expand", "Collapse" and "Break" groups. 
If you would go an extra step, having the group still expandable/collapseable after an entity has been remove and have it as some kind of temporary/local-for-this-case group.
